### PR TITLE
Drop Xcode 14

### PR DESCRIPTION
**Pull request by porter-stripe**
## Motivation
https://jira.corp.stripe.com/browse/MOBILESDK-1810

## Testing
CI

## Changelog
See diff

### CODE REVIEW

File: StripeUICore/StripeUICore/Source/Helpers/StripeUICoreBundleLocator.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Foundation
 
 @_spi(STP) public final class StripeUICoreBundleLocator: BundleLocatorProtocol {
     public static let internalClass: AnyClass = StripeUICoreBundleLocator.self
-    public static let bundleName = "StripeUICore"
+    public static let bundleName = "StripeUICoreBundle"
```
Comment (porter-stripe) [inline StripeUICore/StripeUICore/Source/Helpers/StripeUICoreBundleLocator.swift:14]: We had some install tests failing: https://github.com/CocoaPods/CocoaPods/issues/12051

File: ci_scripts/vm_tools.rb [hunk 2]
```diff
@@ -59,11 +59,11 @@ def build_vm
   rputs 'Building VM images! This will take a while, grab some coffee and keep your Mac awake...'
   setup_vm_requirements
   Dir.chdir("#{Dir.home}/stripe/ios-deploy-vm") do
-    run_command('packer init -var-file="variables.pkrvars.hcl" templates/vanilla-ventura.pkr.hcl')
-    unless `tart list`.include? 'ventura-vanilla'
-      run_command('packer build -var-file="variables.pkrvars.hcl" templates/vanilla-ventura.pkr.hcl')
+    run_command('packer init -var-file="variables.pkrvars.hcl" templates/vanilla-sonoma.pkr.hcl')
```
Comment (davidme-stripe) [inline ci_scripts/vm_tools.rb:62]: We probably don't need to do this just yet (and I haven't added a template for Sonoma), but I'm okay with committing it for now

**Review by davidme-stripe — APPROVED** [review]

**Comment by davidme-stripe** [comment]
LGTM! I'm okay with landing this as-is and doing the "*Bundle" Pod change for the other frameworks in a followup PR, up to you

File: StripeUICore/StripeUICore/Source/Helpers/StripeUICoreBundleLocator.swift (hunk 1 above)
Comment (porter-stripe) [reply StripeUICore/StripeUICore/Source/Helpers/StripeUICoreBundleLocator.swift:14]: Good idea, added!

**Pull request merged**